process/geometry/utils.py

```diff
@@ -0,0 +1,120 @@
+"""
+Module to hold plotting functions, used in plot_proc.py, which are common to multiple reactor components
+"""
+import numpy as np
+
+
+def plotdh(r0: float, a: float, triang: float, kap: float) -> tuple:
+    """Plots half a thin D-section, centred on z = 0
```

Comment:
Does this *plot* the D or does it just return the points to be plot? I cant see any plotting logic